libraries/src/Event/Installer/AfterPackageDownloadFailedEvent.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
```suggestion
 * @copyright  (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
```

This is a new file so the copyright is 2025